src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -186,6 +186,9 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
       adapterConfig.captionsTextTrack1LanguageCode = config.playback.captionsTextTrack1LanguageCode;
       adapterConfig.captionsTextTrack2Label = config.playback.captionsTextTrack2Label;
       adapterConfig.captionsTextTrack2LanguageCode = config.playback.captionsTextTrack2LanguageCode;
+      if (Utils.Object.hasPropertyPath(config.playback, 'options.html5.native.heartbeatTimeout')) {
```

Comment:
merge the entire `options.html5.native` object, like we do in other adapters: 

https://github.com/kaltura/playkit-js-hls/blob/a0627283010f604d47232fff3cecf1e924a4fa6d/src/hls-adapter.js#L177-L179

Reply:
done